dockerfiles/che/launcher.sh

```diff
@@ -300,12 +334,15 @@ update_che_server() {
     CHE_VERSION=${DEFAULT_CHE_VERSION}
   fi
 
-  info "---------------------------------------"
-  info "ECLIPSE CHE: PULLING ${CHE_SERVER_IMAGE_NAME}:${CHE_VERSION} FROM REMOTE REGISTRY"
-  docker pull ${CHE_SERVER_IMAGE_NAME}:${CHE_VERSION} > /dev/null 2>&1
-  info "ECLIPSE CHE: IMAGE ${CHE_SERVER_IMAGE_NAME}:${CHE_VERSION} UPDATED"
-  info "PLEASE RESTART CHE TO COMPLETE THE UPDATE"
-  info "---------------------------------------"
+  CURRENT_IMAGE=$(docker images -q ${CHE_SERVER_IMAGE_NAME}:${CHE_VERSION})
+
+  if [ "${CURRENT_IMAGE}" != "" ]; then 
```

Comment:
I thought about this, but was thinking that `update` would always force an update.  If this is true, then we can move the logic for the pre-check for the image into the launcher_server routine as the only place to check if you have an existing image.  WDYT?